libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/DropInNavigationView.kt

```diff
@@ -92,7 +94,8 @@ class DropInNavigationView @JvmOverloads constructor(
          * Single point of entry for the Mapbox Navigation View.
          */
         attachStarted(
-            DropInNavigationViewCoordinator(navigationContext)
+            DropInNavigationViewCoordinator(navigationContext),
+            DropInInfoPanelCoordinator(navigationContext)
```

Comment:
Try removing this. This is closer to what we did in the `qa-test-app`

Instead, you can add an info panel to each `DropInNavigationViewCoordinator` screen layout. We can still have the `FreeDriveInfoPanelBinder` that is used inside `DropInNavigationViewCoordinator`

```
FreeDriveInfoPanelBinder(navigationViewContext).bind(binding.infoPanel)
```

Reply:
Based on internal discussion with @kmadsen I will refactor this code to use 1 infoPanel per screen so we can test the transitions. If it doesn't work for any reason, we would use just 1 infoPanel for the entire drop in experience.